inst/report/scripts/generate_report.R

```diff
@@ -0,0 +1,705 @@
+# Author(s): Awa Synthia
+# Last modified: 2024
+
+# get fasta of pathogen and/or drug
+#' @export
+get_card_data <- function(pathogen = NULL, drug = NULL) {
```

Comment:
Minor suggestion: we have now switched to camelCase throughout.
https://jravilab.github.io/MolEvolvR/reference/index.html

So, this might be `getCARDdata` or something like that. @the-mayer?

Reply:
done: switched all functions to camelCase